packages/amplify_authenticator/docs/prebuilt-widgets.md

```diff
@@ -0,0 +1 @@
+The amplify_authenticator package provides prebuilt widgets for [building custom UIs](https://ui.docs.amplify.aws/components/authenticator?platform=flutter#full-ui-customization). All of the prebuilt widgets are integrated into the authenticator's state. Forms and form fields will automatically update the authenticator's state on change, and buttons will use data from the state to initiate auth events such as sign in and sign up.
```

Comment:
ultra nit: we need to align on whether 'authenticator' is capitalized in documentation or not. Doesn't need to block this PR.